Justfile

```diff
@@ -31,17 +31,15 @@ lint: clean
       exit 1
     fi
 
-    no_errors_found=true
+    has_errors=false
     for file in $(find hosted -type f -name "*.json"); do
       if ! jq empty $file > /dev/null; then
         printf "[error] %s is not a valid JSON file\n\n" $file
-        no_errors_found=false
+        has_errors=true
       fi
     done
 
-    if [ "$no_errors_found" = true ]; then
-      echo "[success] All JSON files are valid"
-    else
+    if [ "$has_errors" = true ]; then
       exit 1
```

Comment:
maybe just `exit 1` right away instead of assigning to a variable and then reading afterwards?

Reply:
we could! if the user had 3 malformed files then they'd only see one at a time.

this felt more "ergonomic" to show all the files so potentially it could be fixed without having to run it multiple times.

open to either approach!

Reply:
i'm for showing all the files that fail lint at once

Reply:
yeah nice, that's better!